util/setuidgid.c

```diff
@@ -13,6 +13,10 @@
 #define _BSD_SOURCE 1
 #define _DEFAULT_SOURCE 1
 
+/* Otherwise clang-5.0 + Ubuntu 14.-04 complains about an unused macro. */
```

Comment:
I think this is a case where it makes more sense to just turn off that warning in our tests?